src/internal/connector/graph_connector_helper_test.go

```diff
@@ -1013,9 +1013,17 @@ func collectionsForInfo(
 			user,
 			info.category,
 			info.pathElements,
-			false,
-		)
-		mc := mockconnector.NewMockExchangeCollection(pth, len(info.items))
+			false)
+		loc := mustToDataLayerPath(
+			t,
+			service,
+			tenant,
+			user,
+			info.category,
+			info.pathElements,
+			false)
```

Comment:
seems like we could probably leave this out for now if we wanted and just do `pth, pth` in the constructor for the collection?